Tasks/Gradle/sonarqube-common/restresponse.ts

```diff
@@ -0,0 +1,13 @@
+// Data class representing an HTTPS response from the SonarQube server.
```

Comment:
Please use standard class and method comments, which are similar to JavaDoc. Typemap tools and IntelliSense-like tools know how to read them and provide a better expereince in the IDE. See JSDoc for example. VS Code has plugins to generate them.

  /**
     * Description of my method
     * @method myFunction
     * @param {String} myParam
     */